react/components/NewChannelForm/index.js

```diff
@@ -99,7 +99,7 @@ class NewChannelForm extends Component {
     const isDisabled = !(mode === 'resting' || mode === 'error');
 
     return (
-      <Query query={groupsCountQuery}>
+      <Query query={newChannelGroupsQuery}>
```

Comment:
I'd just make this the newChannel query with a fragment for the authorship component in there.